cmd/neofs-node/closer.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+type closer struct {
+	name string
+	fn   func()
+}
+
+func getCloser(c *cfg, name string) *closer {
+	for _, clsr := range c.closers {
+		if clsr.name == name {
+			return &clsr
+		}
+	}
+	return nil
+}
+
+func delCloser(c *cfg, name string) {
+	for i, clsr := range c.closers {
+		if clsr.name == name {
+			c.closers[i] = c.closers[len(c.closers)-1]
```

Comment:
can't say for sure but think that we can rely on the closers' order now or in the future. not sure

Reply:
It is a simplest way to detect do we have closer/worker for the httpcomponent or not. In other case we need to remember id for worker and closer and do not forget to set it to negative value to mark that closer/worker for component deleted or not initialized at that moment, and do not forget to check this when reloading. We have less than 20 workers\closers currently. You prefer this approach?